Mpatel1803/count_Zeros.cpp

```diff
@@ -0,0 +1,48 @@
+//Author: Manav R. Patel ~ patelmanav1803@gmail.com
+#include <iostream>
+using namespace std;
+
+//--Function to count number of zeros--
+int countZeros(int arr[], int size){
+  int count = 0;
+  int temp;
+  int number = size;
+
+//--Sort the array--
+  for(int j=1; j<number; j++){ 
+			  temp = arr[j]; 
+				int i=j;
+				while(i>0 && arr[i-1] >= temp){
+					arr[i] = arr[i-1];
+					--i;
+				}
+				arr[i] = temp;
+	}
+// Count the number of zeros in the array
+  for (int i=0; i<number; i++){
```

Comment:
Would this alone work, without having to sort?

Reply:
Yes it would work without having to sort the numbers